src/views/PersoonDetailView.vue

```diff
@@ -4,36 +4,47 @@
   <utrecht-heading :level="1">Persoonsinformatie</utrecht-heading>
 
   <tab-list v-model="activeTab">
-    <tab-list-data-item label="Contactgegevens" :data="klant">
-      <template #success="{ data }">
-        <klant-details :klant="data" />
+    <tab-list-item label="Contactgegevens">
+      <template #default="{ setError, setLoading }">
+        <klant-details
+          v-if="defaultSysteem"
+          :klant-id="persoonId"
+          :systeem="defaultSysteem"
+          @load="klant = $event"
+          @loading="setLoading"
+          @error="setError"
+        />
       </template>
-    </tab-list-data-item>
+    </tab-list-item>
 
-    <tab-list-data-item label="BRP gegevens" :data="persoon">
-      <template #success="{ data }">
-        <brp-gegevens v-if="data" :persoon="data" />
+    <tab-list-item label="BRP gegevens">
+      <template #default="{ setError, setLoading }">
+        <brp-gegevens
```

Comment:
eerst werd de persoon opgehaald en meegegeven. nu laat je het brp-gegevens component de persoon zelf zoeken. wat is de reden van die wijziging?

Reply:
Dat is in lijn met hoe we het voor contactmomenten/verzoeken doen, en hoe we volgens mij afgesproken hebben dat we het willen: de feature regelt het ophalen van data, en geen gebruik meer maken van de ServiceData abstractie. Dat heb ik destijds alleen voor cm/cv gedaan, en wil ik in de vervolgstory voor zaken doen. Dan werken we toe naar dezelfde oplossing en het verwijderen van de servicedata abstractie.